data/gegno/ships.txt

```diff
@@ -0,0 +1,954 @@
+		hull 22275
+		"outfit space" 815
+		"required crew" 142
+		shields 0
+		"weapon capacity" 412
+		weapon
+			"blast radius" 150
+			"shield damage" 1150
+			"hull damage" 1570
+			"hit force" 1200
+	outfits
+		"Ballistic Cannon" 3
+		"C27 Campaign Core"
+		Hyperdrive
+		"Irate Cannon" 4
+		"R04 Crusade Battery"
+		"RG3 Torch Steering"
+		"RG3 Torch Thruster"
+	engine 0 264
+	gun -17.4 -282 "Irate Cannon"
+	gun 17.4 -282 "Irate Cannon"
+	gun -24.5 -263 "Irate Cannon"
+	gun 24.5 -263 "Irate Cannon"
+	turret 0 -107 "Ballistic Cannon"
+	turret 0 -30.5 "Ballistic Cannon"
+	turret 0 42.5 "Ballistic Cannon"
+	leak leak 50 50
+	leak flame 80 140
+	explode "huge explosion" 18
+	explode "large explosion" 30
+	explode "medium explosion" 46
+	explode "small explosion" 40
+	explode "tiny explosion" 80
+	"final explode" "final explosion large"
+	description "Gneiss Battleships are the Gegno Vi's true sword & shield in war. They are slightly older than the Hornfels and Schists, but carry the heaviest armament, bolstering 3 huge mounts dedicated to hold enormous Ballistic Cannons. Gniesses also have extremely strong armor plating, and can take a hefty beating while firing dangerously threatening broadsides at the opposing force."
+	description `	Unfortunately, in exchange for huge firepower and defensive armor, they lack speed and maneuverability, even when fitted with the largest engine systems the Vi have to offer.`
+
+ship Protolith
+	sprite ship/protolith
+	thumbnail thumbnail/protolith
+	swizzle 0
+	attributes
+		category "Heavy Freighter"
+		licenses
+			"Gegno Civilian"
+			"Vi Evocati"
+			"Vi Centurion"
+			"Vi Lord"
+		cost 17600000
+		mass 1260
+		bunks 292
+		"cargo space" 530
+		drag 16.1
+		"engine capacity" 176
+		"fuel capacity" 700
+		"heat dissipation" 0.4
+		hull 19575
+		"outfit space" 568
+		"required crew" 84
+		shields 0
+		"weapon capacity" 180
+		weapon
+			"blast radius" 280
+			"shield damage" 1550
+			"hull damage" 2050
+			"hit force" 3950
+	outfits
+		"C27 Campaign Core"
+		"Chloeric Turret" 2
+		Hyperdrive
+		"Irate Cannon" 2
+		"R02 Battlezone Battery"
+		"R03 Warforge Battery"
+		"RG3 Torch Steering"
+		"RG3 Torch Thruster"
+	engine 0 260
+	engine -39 250
+	engine 39 250
+	gun 37 -212 "Irate Cannon"
+	gun 45 -199.5 "Irate Cannon"
+	turret 0 -33 "Chloeric Turret"
+	turret 0 8 "Chloeric Turret"
+	leak leak 30 50
+	leak flame 30 80
+	leak "big leak" 50 30
+	explode "huge explosion" 30
+	explode "large explosion" 45
+	explode "medium explosion" 35
+	explode "small explosion" 25
+	"final explode" "final explosion large"
+	description "Protoliths are descendants of the Gegno's early City-Ships used to colonize nearby planets. They now carry hundreds of battle-ready Vi both in and out of war, and carry huge cargo loads to supply depots. They are not typically used as warships but can be mounted with a small payload that can threaten most ships smaller than them, or fend off larger ships long enough for reinforcements to arrive."
```

Comment:
A lot of these ships have excess energy but will overheat if you even think about firing the weapons. Can they get some low energy efficiency active cooling systems or something?
Also, there's a lot of empty space on this ship. Can it have bigger guns?
![image](https://user-images.githubusercontent.com/20605679/173263361-05a2386e-d54a-4923-b1fd-773514713508.png)

Reply:
If the Vi designers did not intend for it to carry larger weapons, it should have less weapon capacity. Maybe 130 or 140.

If this unused space is a reflection of their technology being worse when it was designed and so weapons of the same performance took up more space but their modern technology is more compact and they just haven't designed a newer ship to better take advantage of those improvements, it's fine, though I would suggest adding cargo expansions to show this effect in action. Maybe they've got updated versions of this design in the works that would reduce weapon capacity and divert some to engine capacity, cargo soace, and/or bunks.

Regardless, this ship absolutely does not need a C27, it doesn't even need a C17. A couple of C7s and the 23t battery should be plenty.